Description: Fixes tools/pip-requires which declares == when >= is actually ok
Author: Ghe Rivero <ghe@debian.org>
Forwarded: not-needed

--- a/tools/pip-requires
+++ b/tools/pip-requires
@@ -1,6 +1,6 @@
 # keystone dependencies
-pam==0.1.4
-WebOb==1.0.8
+pam>=0.1.4
+WebOb>=1.0.8
 eventlet
 greenlet
 PasteDeploy
